etcdserver/etcdserverpb/rpc.proto

```diff
@@ -787,6 +795,15 @@ message DefragmentResponse {
   ResponseHeader header = 1;
 }
 
+message MoveLeaderRequest {
+  // target is the node ID for the new leader.
+  uint64 target = 1;
```

Comment:
judging by the rest of rpc.proto, this probably needs `ID` in the name to be consistent with other rpc message naming-- something like targetID, memberID, or ID